### ISSUE TITLE
Field name and value

### ISSUE THREAD

**Issue by MikeBishop** [httpwg/admin#54]
We explicitly say not to \<tt> / \` field names on reference.  Do we want that enclosure when we're explicitly citing a field name and value pair?  That is:

> ...by sending a `Vary: Client-Cert` response header.

or

> ...by sending a "Vary: Client-Cert" response header.

or 

> ...by sending a Vary: Client-Cert response header.

I'm inclined to think that quotes are the most clear, since they appear in the text output, but ignoring text output my preference would be for \<tt>

**Comment by mnot** [comment]
Yeah, in a perfect world, we'd use `tt`, but the text output is not great. Personally, I'd lean towards quotes, although that may result in some confusing constructs (eg if the field value ends in quotes, or contains lots of them).

**Comment by MikeBishop** [comment]
From a cursory search of the RFC 911X documents, it appears that full fields (name + value) are usually set out as an illustration similar to `http-message` objects.  The exception I've found so far was:

> Because the meaning of "Pragma: no-cache" in responses was never specified, it does not provide a reliable replacement for "Cache-Control: no-cache" in them.

So obviously we went quotes there.

**Comment by mnot** [comment]
That seems like a reasonable rule of thumb: examples should generally be in http-message sections, unless they're so simplie that they're unambiguous in quotes.
